modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go

```diff
@@ -101,8 +101,8 @@ func (o *OrgFormModal) SetProps() {
 				"required":  true,
 				"rules": []interface{}{
 					map[string]interface{}{
-						"msg":     "由小写字母、数字，-组成",
-						"pattern": "/^[a-z0-9-]*$/",
+						"msg":     "由小写字母、数字，-组成(不允许纯数字)",
+						"pattern": "/^(?![0-9]+$)[a-z0-9-]*$/",
```

Comment:
is empty string allowed?

Reply:
not allowed
![image](https://user-images.githubusercontent.com/30427474/124857676-531c5980-dfdf-11eb-8012-09389fc848e4.png)
and backend will check org name